### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- #244

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -10,31 +10,21 @@ source _common.sh
...
-
 #=================================================
 # STANDARD UPGRADE STEPS
 #=================================================
 # STOP SYSTEMD SERVICE
 #=================================================
 ynh_script_progression --message="Stopping a systemd service..."
 
-ynh_systemd_action --service_name=$app --action="stop" --log_path="systemd" --line_match="Stopping Vaultwarden Server"
+ynh_systemd_action --service_name=$app --action="stop" --log_path="systemd" --line_match="Vaultwarden process exited"
```
Comment (tituspijean) [inline scripts/upgrade:20]: It looks like the service does not print the line_match text in versions before 1.31.0, which induces a mandatory long wait for the timeout.

**Comment by kay0u** [comment]
Tested @ home

**Comment by vwvw** [comment]
Likely necessary to fix #266

**Comment by Tagadda** [comment]
The wierd thing now is that we have merged this PR that requires YunoHost 12.x, but it's not out yet ^^
Now the catalog complains that the app is broken...

**Comment by kay0u** [comment]
Because the stable version of the CI is still bullseye

**Pull request merged**